### PR TITLE
Order insights by published at

### PR DESCRIPTION
**Pull request by tspenov**

### CODE REVIEW

File: test/sanbase_web/graphql/post_test.exs [hunk 1]
```diff
@@ -274,14 +274,14 @@ defmodule SanbaseWeb.Graphql.PostTest do
     assert result |> hd() |> Map.get("text") == post.text
   end
 
-  test "getting all posts ranked", %{user: user, user2: user2, conn: conn, poll: poll} do
+  test "getting all insights ordered", %{user: user, user2: user2, conn: conn, poll: poll} do
```
Comment (kalkov) [inline test/sanbase_web/graphql/post_test.exs]: A separate test just for asserting ordering will guarantee us that will always work as expected. 
Example: Create 3 posts with custom `created_at` values, set `published_at` and assert the ordering. 
Could be in another PR if this one is urgent.

Comment (tspenov) [reply test/sanbase_web/graphql/post_test.exs]: sounds good - will add one
